corehq/apps/app_manager/management/commands/get_domains_with_itext_usage.py

```diff
@@ -18,7 +18,7 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument(
             'itext_type',
-            help='audio, expanded-audio, video, etc.',
```

Comment:
@biyeun This command looks [pretty old](https://github.com/dimagi/commcare-hq/pull/17612), any idea if it's still useful?

Reply:
@orangejenny wow yeah this is pretty old. i don't think we're using it anymore too...this feels like it was a one-off product request to get some data

Reply:
@djmore9 does this ring a bell for you at all?

Reply:
@biyeun cool, let's kill it :)  I'm on a little bit of a rampage lately against old management commands, just because they pop up in PRs like this. @michelleliang1  could you delete this file altogether?

Reply:
@orangejenny @biyeun file deleted!